app/livechat/server/hooks/beforeDelegateAgent.js

```diff
@@ -5,18 +5,20 @@ import { Users, LivechatDepartmentAgents } from '../../../models';
 
 callbacks.add('livechat.beforeDelegateAgent', (options = {}) => {
 	const { department, agent } = options;
+	if (!settings.get('Livechat_kill_switch')) {
```

Comment:
### How about reducing code blocks nesting _(return early)_?

sample code for reference:

```
if (settings.get('Livechat_kill_switch')) {
	return null;
}
if (agent) {
	return agent;
}
if (!settings.get('Livechat_assign_new_conversation_to_bot')) {
	return null;
}
if (department) {
	return LivechatDepartmentAgents.getNextBotForDepartment(department);
}
return Users.getNextBotAgent();
```